src/components/cart/CartSidebar/cart-sidebar.scss

```diff
@@ -30,12 +30,8 @@
   padding: var(--space-5) var(--page-padding-phone) var(--space-4);
   background-color: var(--bg-neutral-lightest);
 
-  button {
-    position: absolute;
-    right: calc(-1 * var(--page-padding-phone));
-    margin-right: var(--space-0);
-    color: var(--color-text);
-    background-color: transparent;
+  .cart-sidebar__button {
+    margin-right: calc(-1 * var(--space-1));
```

Comment:
Is this negative margin intentional?

https://user-images.githubusercontent.com/381395/151890587-61f1586c-b5e4-4d51-851e-70e68709af51.mp4